exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/BulkIndexRequest.java

```diff
@@ -74,16 +73,14 @@ void index(
 
   private static byte[] serializeRecord(final Record<?> record, final RecordSequence recordSequence)
       throws IOException {
-    final var serializedRecord = MAPPER.writer().writeValueAsBytes(record);
-
-    // Enhance the serialized record by its sequence number. The sequence number is not a part of
-    // the record itself but a special property for Elasticsearch. It can be used to limit the
-    // number of records when reading from the index, for example, by using a range query. Read
-    // https://github.com/camunda/zeebe/issues/10568 for details.
-    final var enhancedRecord = MAPPER.readValue(serializedRecord, MAP_TYPE_REFERENCE);
-    enhancedRecord.put(RECORD_SEQUENCE_PROPERTY, recordSequence.sequence());
-
-    return MAPPER.writer().writeValueAsBytes(enhancedRecord);
+    return MAPPER
+        .writer()
+        // Enhance the serialized record by its sequence number. The sequence number is not a part
+        // of the record itself but a special property for Elasticsearch. It can be used to limit
+        // the number of records when reading from the index, for example, by using a range query.
+        // Read https://github.com/camunda/zeebe/issues/10568 for details.
+        .withAttribute(RECORD_SEQUENCE_PROPERTY, recordSequence.sequence())
```

Comment:
:+1: Wow. It's magic. :magic_wand: 

I was looking for something like that but I wasn't aware of this feature. It's a perfect fit here. :dart: